CHANGELOG.md

```diff
@@ -1,10 +1,172 @@
 TODO: This document was manually maintained so might be incomplete. The
 automation effort is tracked in
-https://github.com/kubernetes/client-go/issues/234.
+https://github.com/kubernetes/test-infra/issues/5843.
 
 Changes in `k8s.io/api` and `k8s.io/apimachinery` are mentioned here
 because `k8s.io/client-go` depends on them.
 
+# v8.0.0
+
+**Breaking Changes:**
+
+* `KUBE_API_VERSIONS` has been removed. Pleas use the `runtime-config` flag instead.
+
+    * [https://github.com/kubernetes/kubernetes/pull/63165](https://github.com/kubernetes/kubernetes/pull/63165)
+
+* The shortcuts which were moved server-side in v1.9 have been now moved to client-go, instead of being hardcoded in kubectl.
```

Comment:
the actual change here is: the client-go/discovery RESTMapper has been moved to client-go/restmapper.

Reply:
fixed